src/traceability/azure-traceability-helper.ts

```diff
@@ -120,7 +121,7 @@ async function createDeploymentReport(kubectl: Kubectl, deploymentConfig: Deploy
   });
 
   const deploymentReport: DeploymentReport = new DeploymentReport();
-  await deploymentReport.initialize(InputParameters.githubToken, artifacts, 'succeeded', resource);
+  await deploymentReport.initialize(InputParameters.githubToken, artifacts, status, resource);
```

Comment:
Is the expected string literal 'succeeded' or 'success' ? (It is set as 'success' in run())